sdk/state/integrationtests/state_test.go

```diff
@@ -158,26 +138,17 @@ func TestOpenUpdatesUncoordinatedClose(t *testing.T) {
 		t.Log("Current channel iteration numbers: I: ", sendingChannel.NextIterationNumber(), "R: ", receivingChannel.NextIterationNumber())
 		t.Log("Proposal: ", i, paymentLog, amount/1_000_0000)
 
-		// Sender: creates new Payment, sends to other party
+		// Sender: creates new Payment, signs, sends to other party
 		payment, err := sendingChannel.ProposePayment(amount)
 		require.NoError(t, err)
 
-		var authorized bool
-
-		// Receiver: receives new payment, validates, then confirms by signing both
-		payment, authorized, err = receivingChannel.ConfirmPayment(payment)
-		require.NoError(t, err)
-		require.False(t, authorized)
-
-		// Sender: re-confirms P_i by signing D_i and sending back
-		payment, authorized, err = sendingChannel.ConfirmPayment(payment)
+		// Receiver: receives new payment, validates, then confirms by signing
```

Comment:
maybe the biggest nit of all time, but my OCD kicked in and I notice sometimes on comments we end with a period and other times we don't. We should probably choose for consistency sake. Periods always? (looks like that's most of the comments, I may have been the one forgetting) 😄

Reply:
+1. I'd prefer capitalized with periods. Then comments are consistent regardless of whether they are single sentence or multiple sentence. They're also consistent with comments on functions. Let's do a pass over the repo and fix them once most of the existing PRs are done (to reduce conflicts).

Reply:
sgtm!

Reply:
I've created https://github.com/stellar/experimental-payment-channels/issues/193 to track this effort.

Reply:
I think 193 is tracking a different thing. But we could probably just do it in the same PR 😄 . The comment changes will be pretty straightforward

Reply:
Haha. I got a bit lost didn't I. I'm not sure where our conversation was about adding bad actor testing, but the comment was intended to land there 😄.